soc/nordic/sysbuild.cmake

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if(SB_CONFIG_VPR_LAUNCHER)
+  set(launcher_core "cpuapp")
+  string(REPLACE "/" ";" launcher_quals ${BOARD_QUALIFIERS})
+  list(GET launcher_quals 1 launcher_soc)
+  string(CONCAT launcher_board ${BOARD} "/" ${launcher_soc} "/" ${launcher_core})
+
+  set(image vpr_launcher)
+  ExternalZephyrProject_Add(
+    APPLICATION ${image}
+    SOURCE_DIR ${ZEPHYR_BASE}/samples/basic/minimal
+    BOARD ${launcher_board}
+  )
+
+  if(SB_CONFIG_VPR_LAUNCHER_XIP)
+    sysbuild_cache_set(VAR ${image}_SNIPPET APPEND REMOVE_DUPLICATES nordic-ppr-xip)
+  else()
+    sysbuild_cache_set(VAR ${image}_SNIPPET APPEND REMOVE_DUPLICATES nordic-ppr)
+  endif()
```

Comment:
This won't work.

You're not allowed to change snippets between incremental builds, build folder must be cleaned pristinely. 

But the sysbuild menuconfig would allow you to change the value of `SB_CONFIG_VPR_LAUNCHER_XIP` between builds.